x-pack/plugins/security_solution/common/endpoint/data_loaders/utils.ts

```diff
@@ -18,3 +20,13 @@ export const wrapErrorIfNeeded = (error: Error): EndpointDataLoadingError =>
 
 // Use it in Promise's `.catch()` as `.catch(wrapErrorAndRejectPromise)`
 export const wrapErrorAndRejectPromise = (error: Error) => Promise.reject(wrapErrorIfNeeded(error));
+
+export const mergeAndAppendArrays = <T, S>(destinationObj: T, srcObj: S) => {
```

Comment:
can you add the return type to this function? I think it should be `T`

Reply:
okie